instrumentation/net_http/opentelemetry-instrumentation-net_http.gemspec

```
@@ -25,13 +25,13 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
   spec.required_ruby_version = '>= 2.5.0'
 
-  spec.add_dependency 'opentelemetry-api', '~> 1.0.0.rc3'
+  spec.add_dependency 'opentelemetry-api', '~> 1.0'
   spec.add_dependency 'opentelemetry-common', '~> 0.19.1'
```

```suggestion
  spec.add_dependency 'opentelemetry-common', '~> 0.19.2'
```
I think this needs to be applied "everywhere" as well.